include/integration/simple_MC_integration.hpp

```diff
@@ -0,0 +1,194 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2021 Vissarion Fisikopoulos
+// Copyright (c) 2018-2021 Apostolos Chalkis
+
+// Contributed and/or modified by Suraj Choubey, as part of Google Summer of Code 2021 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef SIMPLE_MC_INTEGRATION_HPP
+#define SIMPLE_MC_INTEGRATION_HPP
+
+#include <iostream>
+#include <fstream>
+#include <random>
+#include <vector>
+#include "convex_bodies/hpolytope.h"
+#include "Eigen/Eigen"
+#include "generators/known_polytope_generators.h"
+#include "boost_random_number_generator.hpp"
+#include "cartesian_geom/cartesian_kernel.h"
+#include "random_walks/random_walks.hpp"
+#include "volume/volume_sequence_of_balls.hpp"
+#include "volume/volume_cooling_gaussians.hpp"
+#include "volume/volume_cooling_balls.hpp"
+#include "misc.h"
+
+typedef double NT;
+typedef Cartesian<NT> Kernel;
+typedef typename Kernel::Point Point;
+typedef std::vector<Point> Points;
+typedef HPolytope<Point> HPOLYTOPE;
+typedef boost::mt19937 RNGType;
+typedef BoostRandomNumberGenerator<RNGType, NT> RandomNumberGenerator;
+typedef typename HPolytope<Point>::MT MT;
+typedef typename HPolytope<Point>::VT VT; 
+
+typedef const unsigned int Uint;  // positive constant value for no of samples & dimensions
+enum volType { CB , CG , SOB }; // Volume type for polytope
+
+// To check if two n-dimensional points ensure valid limits in integration 
+bool validLimits(Point LL, Point UL){
```

Comment:
Should this method throw an exception instead of giving an output to stdout?

Reply:
This is meant to check if UL[i] >= LL[i]

Reply:
Thank you! What I meant was that these errors should be printed on stderr, which you do below.